Atlassian.Jira/IIssueTypeService.cs

```diff
@@ -1,25 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Atlassian.Jira
-{
-    /// <summary>
-    /// Represents the operations on the issue types of jira.
-    /// Maps to https://docs.atlassian.com/jira/REST/latest/#api/2/issuetype.
-    /// </summary>
-    public interface IIssueTypeService
-    {
-        /// <summary>
-        /// Returns all the issue types within JIRA.
-        /// </summary>
-        Task<IEnumerable<IssueType>> GetIssueTypesAsync(CancellationToken token = default(CancellationToken));
-
-        /// <summary>
-        /// Returns the issue types within JIRA for the project specified.
-        /// </summary>
-        Task<IEnumerable<IssueType>> GetIssueTypesForProjectAsync(string projectKey, CancellationToken token = default(CancellationToken));
```

Comment:
why delete this?
any divergence from the base package will make it harder to merge if/when we will want to do it

Reply:
I'm fine with the inconsistency of the main repo, I think it shouldn't have been here in the first place, as the interface's documentation states above `Maps to https://docs.atlassian.com/jira/REST/latest/#api/2/issuetype`, this endpoint is not `api/2/issuetype` but `api/2/project` which should be in `IProjectService` anyway